### PR TITLE
Sandbox

### PR DESCRIPTION
**Pull request by andreyklumchyk**
#### Contains:  

- documentation and some workflow rules;  
- deployment instruction for [Heroku](heroku.com) service;  
- base architecture of future application with prepared Docker image;  
- base CI jobs like tests & docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,409 @@
...
+
+
+## Implementation requirements
+
+1. Code should be documented with [phpdoc] and is able to generate documentation in a form of static website (you may deploy it with site or on [GitHub Pages] as well).   
+2. Code should be covered with unit tests.
+3. [E2E (end-to-end) tests][2] should cover all the required features.
+4. Frontend must work with backend throw [GraphQL]. 
+5. Backend templates must use [Twig].  
+6. Backend architecture should be [SOLID].
```
Comment (tyranron) [inline README.md]: SOLID is not an architecture pattern or a pattern at all. It's a set of design principles.

File: README.md [hunk 2]
```diff
@@ -1 +1,409 @@
...
+4. Show messages history with specified account.
+
+
+
+## Implementation requirements
+
+1. Code should be documented with [phpdoc] and is able to generate documentation in a form of static website (you may deploy it with site or on [GitHub Pages] as well).   
+2. Code should be covered with unit tests.
+3. [E2E (end-to-end) tests][2] should cover all the required features.
+4. Frontend must work with backend throw [GraphQL].
```
Comment (tyranron) [inline README.md]: `throw` means this: `(╯°□°）╯︵ ┻━┻`

File: README.md [hunk 3]
```diff
@@ -1 +1,409 @@
...
+
+Firstly, you should demonstrate that Chat application is working without registration. Next register two account that they could found each other and chat.
+
+Please, before demonstration make sure that your [Heroku] instance is not [sleeping][10].
+
+
+
+## Final design of application
+
+Final design of the implemented application may vary from the provided one. The provided design aims only to explain the expected result better.
```
Comment (tyranron) [inline README.md]: It's unclear what design we're talking about here.

**Review by tyranron — APPROVED** [review]

**Comment by andreyklumchyk** [comment]
Create a simple PHP sandbox with requirements for chat application

**Pull request merged**